Resources/Maps/_NF/Shuttles/bazaar.yml

```
@@ -26,7 +26,7 @@ entities:
       name: Bazaar
     - type: Transform
       pos: -8.499361,3.2310195
-      parent: invalid
+      parent: 963
```

```suggestion
      parent: invalid
```
This is your grid, it shouldn't be parented to anything when you save it as the grid (it is the world).